constants/organizations.ts

```diff
@@ -0,0 +1,36 @@
+const organizations = [
+  {
+    id: 1,
+    image: "/images/organizations/Crime-Junkie-NPO_DNASolves.png",
+    alt: "Organization 1",
+    websiteURL: "https://dnasolves.com/",
+  },
+  {
+    id: 2,
+    image:
+      "/images/organizations/Crime-Junkie-NPOs_Center-for-Survivor-Agency-and-Justice.png",
+    alt: "Organization 2",
+    websiteURL: "https://csaj.org/",
+  },
+  {
+    id: 3,
+    image: "/images/organizations/Crime-Junkie-NPOs_DNA-Doe-Project.png",
+    alt: "Organization 3",
+    websiteURL: "https://dnadoeproject.org/",
+  },
+  {
+    id: 4,
+    image:
+      "/images/organizations/Crime-Junkie-NPOs_Elizabeth-Smart-Foundation.png",
+    alt: "Organization 4",
+    websiteURL: "https://www.elizabethsmartfoundation.org/",
+  },
+  {
+    id: 5,
+    image: "/images/organizations/Crime-Junkie-NPOs_HSI.png",
+    alt: "Organization 5",
+    websiteURL: "https://www.hsi.org/",
+  },
+]
+
+export default organizations
```

Comment:
would be better to have this inside `config` folder, not `constants`. i can give you some examples tomorrow if you need, when and what goes where :)